b2share/modules/records/triggers.py

```diff
@@ -82,3 +98,17 @@ def index_record(record):
         version=record.revision_id,
         version_type='external_gte',
     )
+
+
+def check_record_immutable_fields(record):
```

Comment:
You're right, these should be permissions. I will remove them once we have that module; or could you remove them in the same commit?